### ISSUE TITLE
Labels for Object List and People List dropdowns are insufficient for screen reader

### ISSUE THREAD

**Issue by emclaren** [mozilla/hubs#2203]
**Description**
Labels for Object List and People List are insufficient for screen reader user
Right now screen reader only hears "6" and "3" to describe the UI elements. We should prepend better labels in front of each item such as "Objects 6" and "People 3"

![image(5)](https://user-images.githubusercontent.com/6835324/76281725-2c1b0f80-6253-11ea-90fe-94d66a76ace2.png)

**Additional context**
Flagged by the a11y community using Hubs
